bazel/repository_locations.bzl

```diff
@@ -9,11 +9,11 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/bazelbuild/bazel-toolchains/archive/0.28.5.tar.gz"],
     ),
     boringssl = dict(
-        # Use commits from branch "chromium-stable-with-bazel"
-        sha256 = "18edf961f8377e8d10fd8497bc8a331def9cb60a6c2a50a4c8eb322b045042d5",
-        strip_prefix = "boringssl-87d1c8f292e5184fd727efe84f458d89687d7742",
-        # chromium-76.0.3809.87
-        urls = ["https://github.com/google/boringssl/archive/87d1c8f292e5184fd727efe84f458d89687d7742.tar.gz"],
+        # Use commits from branch "master-with-bazel"
+        sha256 = "c712766ddc844de2a38e686e1cdd7288795e9a6fe7f699c6636f1b76703db84e",
+        strip_prefix = "boringssl-265728decec4370cd02b941f72fba9f0735e2923",
+        # chromium-77.0.3865.37 (BETA)
```

Comment:
Can you elaborate how to pick a commit that is being used in specific version of chrome/chromium?

Reply:
Added, PTAL.